src/common/UploadOptions.svelte

```diff
@@ -1,64 +1,118 @@
 <script>
-  import Omniselect from "./Omniselect";
-  import Checkbox from "./Checkbox";
   import {
     defaultLanguage,
     languages,
     textractLanguages,
-  } from "@/api/languages";
```

Comment:
This is really just my own preference here. I'd rather have full file extensions, because it makes it easier to know what I'm importing -- a JS file or a Svelte component or something else.